src/components/Input/BoxedInput.ts

```diff
@@ -47,6 +47,11 @@ const BoxedInput: FC<InternalInputComponentProps> = styled(BaseInput)`
         ${p => (p.hasValue || p.placeholder ? activeBoxedPosition(p.size) : null)};
         color: ${getLabelColor};
         background: ${p => (p.inverted ? Colors.AUTHENTIC_BLUE_900 : Colors.WHITE)};
+        background: ${p =>
+            `linear-gradient(0deg, ${p.inverted ? Colors.AUTHENTIC_BLUE_900 : Colors.WHITE} calc(50% + ${
+                // @ts-ignore
+                p.size === 'small' ? '0.0825rem' : '0.0625rem'
+            }), transparent 50%)`};
```

Comment:
I think the `background` property above (line 49) is going to be overwritten by this one and ignored.

Reply:
Yes, that's the idea. Although we don't enforce to support legacy browsers, I found it convenient to apply these "hacky" solutions as a sort of incremental enhancement.